lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -149,6 +148,24 @@ def config_content
           escape_and_echo(client_rb)
         end
 
+        def get_log_location
+          if @chef_config[:config_log_location].equal?(:win_evt)
+            return %Q{log_location  :#{@chef_config[:config_log_location]}\n}
+          elsif @chef_config[:config_log_location].equal?(:syslog)
+            raise "syslog is not supported for log_location on Windows OS\n"
+          elsif (@chef_config[:config_log_location].equal?(STDOUT))
+            return "log_location    STDOUT\n"
+          elsif (@chef_config[:config_log_location].equal?(STDERR))
+            return "log_location    STDERR\n"
+          elsif @chef_config[:config_log_location].nil? || @chef_config[:config_log_location].empty?
+            return "log_location    STDOUT\n"
+          elsif @chef_config[:config_log_location]
+            return %Q{log_location  "#{@chef_config[:config_log_location]}"\n}
+          else
+            return "log_location    STDOUT\n"
```

Comment:
being the last statement in the method, you shouldn't need to use `return` anywhere here. Only one conditional be matched and we would return the string.

Given that you always return "log_location<tab>", I would have this method only return the value portion of that line and around line 87 use something like:

```
client_rb << "log_location\t#{get_log_location}"
```